cmd/opm/alpha/veneer/basic.go

```diff
@@ -21,10 +21,19 @@ func newBasicVeneerRenderCmd() *cobra.Command {
 	)
 	cmd := &cobra.Command{
 		Use:   "basic basic-veneer-file",
-		Short: "Generate a declarative config blob from a single 'basic veneer' file",
-		Long:  `Generate a declarative config blob from a single 'basic veneer' file, typified as a declarative configuration file where olm.bundle objects have no properties`,
-		Args:  cobra.ExactArgs(1),
+		Short: "Generate a declarative config blob from a single 'basic veneer' file \nWhen FILE is '-' or not provided, the veneer is read from standard input",
+		Long:  "Generate a declarative config blob from a single 'basic veneer' file, typified as a declarative configuration file where olm.bundle objects have no properties \nWhen FILE is '-' or not provided, the veneer is read from standard input",
```

Comment:
Two in one suggestion:
1. You can use backticks and actual newlines in the code (rather than escape newline characters), which can help with readability.
2. Drop the mention of FBC, minus bundle properties. re, my previous comment.

```suggestion
		Long:  `Generate a declarative config blob from a single 'basic veneer' file.
		
When FILE is '-' or not provided, the veneer is read from standard input.`,
```